tests/gen_pickles.py

```diff
@@ -21,7 +21,7 @@ def write(name, proto):
 def generate():
     if not USE_CYTHON:
         raise RuntimeError("Cython is required")
-    for proto in range(pickle.HIGHEST_PROTOCOL):
+    for proto in range(pickle.HIGHEST_PROTOCOL+1):
```

Comment:
please use spaces around operators.

Reply:
flake8 was alright with that version.
I don't mind the change but do you have any refs to support spacing?

Reply:
It's more readable and a common standard to put spaces around operator (except in kwargs in function calls)

Reply:
For example, https://www.python.org/dev/peps/pep-0008/#id28

Reply:
Any reason it's not merged yet?

Reply:
Just that we're all busy humans :)